sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AADWebAppConfiguration.java

```diff
@@ -86,7 +86,7 @@ private AzureClientRegistration createDefaultClient() {
         return new AzureClientRegistration(client, accessTokenScopes);
     }
 
-    private int resourceServerCount(Set<String> scopes) {
+    public static int resourceServerCount(Set<String> scopes) {
```

Comment:
I think only the number of http is not accurate enough, we should count the number of unique urls.